backend/data/processors/conditions_preprocessing.py

```diff
@@ -266,6 +265,7 @@ def convert_AND_OR(processed):
 def convert_coreqs(processed):
     """ Puts co-requisites inside square brackets """
     processed = processed.rstrip()
+    processed = re.sub(r"concurrently?;?:?\s?(.*)", r"[\1]", processed, flags=re.IGNORECASE)
```

Comment:
LMAO

Reply:
Yeah literally copy paste the coreq line haha